Comments and documentation (#4047)
feldera/feldera: docs/pipelines/fault-tolerance.md

are disabled, then the user should occasionally invoke this API to
ensure that the checkpoint feature is useful.

> ⚠️ Writing a checkpoint should ordinarily be a fairly fast operation

> that takes a second or less.  However, the [Delta Lake input

Reviewer: ```suggestion
that takes several seconds.  However, the [Delta Lake input
```

even for small pipelines, I see checkpoint times in the order of 10s. I don't think I've ever seen it complete in <1s.
